### PR TITLE
feat: add global custom icon transformation

### PR DESCRIPTION
**Pull request by userquin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -547,6 +547,25 @@ import IconBar from '~icons/my-yet-other-icons/bar'
 > - To make your icons color adaptable, set `fill="currentColor"` or `stroke="currentColor"` in your SVG.
 > - Leave the `height` and `width` unspecified, we will set them for you.
 
+### Global Custom Icon Transformation
+
+From version `0.14.2`, when loading your custom icons, you can transform them, for example adding `fill` attribute with `currentColor`:
+```ts
+UnoCss({
```
Comment (antfu) [inline README.md:554]: You are using the wrong example? 👀

Comment (userquin) [reply README.md:554]: why? do you mean the `custom` icon not present in previous custom collections?

Comment (antfu) [reply README.md]: This is unocss or unplugin-icons?

Comment (userquin) [reply README.md]: :rofl:

**Pull request merged**